app/userland/settings/js/views/general.js

```diff
@@ -484,6 +545,35 @@ class GeneralSettingsView extends LitElement {
     this.requestUpdate()
   }
 
+  onSearchEngineChange (e) {
+    const newValue = e.target.value
+    this.settings.selected_search_engine = parseInt(newValue)
+    beaker.browser.setSetting('selected_search_engine', this.settings.selected_search_engine)
+    this.requestUpdate()
```

Comment:
Can you render a "Settings Updated" toast here? (See the other update handlers for how)

Reply:
Sure!